### ISSUE TITLE
Helm charts v4.4.11 and v4.4.12 are missing

### ISSUE THREAD

**Issue by dnskr** [minio/operator#1058]
## Expected Behavior
Latest versions of operator and tenant helm charts should be released and available to install

## Current Behavior
```
$ helm repo up
Hang tight while we grab the latest from your chart repositories...
...Successfully got an update from the "minio" chart repository
Update Complete. ⎈Happy Helming!⎈

$ helm search repo
NAME                            CHART VERSION   APP VERSION     DESCRIPTION
minio/minio-operator            4.3.7           v4.3.7          A Helm chart for MinIO Operator
minio/operator                  4.4.10          v4.4.10         A Helm chart for MinIO Operator
minio/tenant                    4.4.10          v4.4.10         A Helm chart for MinIO Operator
```

## Possible Solution
Publish helm charts and update index file

**Comment by dvaldivia** [comment]
I'll fix this in the next release, I've been forgetting to generate those files

**Comment by dnskr** [comment]
Closed because minio/operator:4.4.13 and minio/tenant:4.4.13 has been released
